### ISSUE TITLE
Existing material review

### ISSUE THREAD

**Issue by svenvanderburg** [esciencecenter-digital-skills/lesson-machine-learning-intro#5]

**Comment by svenvanderburg** [comment]
# Review
## Introduction to Machine Learning with scikitlearn
https://carpentries-incubator.github.io/machine-learning-novice-sklearn/

* **More about understanding than applying** Compared to our SICS-Odissei lesson this is a bit more about understanding machine learning than applying it with scikitlearn. For example, we first code up a least squares regression ourselves, before starting to use scikitlearn. I quite like this, but it limits you to cover less powerful machine learning methods.
* **Neural networks with scikitlearn** I would say the episode on neural networks would fit better in a deep learning lesson.
* **Misses machine learning validation** There is not much about validation of your approach
* **Quite heavy on the unsupervised learning**

## Introduction to Machine Learning in Python
* **Very heavy on understanding instead of applying** Everything is coded

**Comment by CunliangGeng** [comment]
It would cost more time/resource to merge lessons than just incubating our own star🌟 lesson. Also, we can fully control the direction of our own lesson, e.g. adapting it to meet new demands of our partners. 

The [ML novice sklearn](https://carpentries-incubator.github.io/machine-learning-novice-sklearn/) lesson was [started from 2013](https://github.com/carpentries-incubator/machine-learning-novice-sklearn/graphs/code-frequency), with rare commits to it in the past 9 years(!!!), and now it's still in alpha version. It's hard to convince me that it's a popular or successful lesson. Then why should we invest in it?

**Comment by svenvanderburg** [comment]
@CunliangGeng those old commits are from the carpentries lesson template, development on the ML lesson has started in 2019 (see [commits](https://github.com/carpentries-incubator/machine-learning-novice-sklearn/commits/gh-pages?after=5c391e09fc718077d1bcb814db2a1529d844a3cc+139&branch=gh-pages&qualified_name=refs%2Fheads%2Fgh-pages)).

But you are right the development of the lesson is not super active, and it is in alpha for quite long now.

I decided to go ahead with porting our materials to carpentries-style lesson. But I also reached out to the contributors of the ML novice sklearn lesson. See more details in https://github.com/carpentries-incubator/machine-learning-novice-sklearn/issues/37
